ios/RNCWKWebView.m

```diff
@@ -540,6 +549,13 @@ - (void)setScrollEnabled:(BOOL)scrollEnabled
 {
   _scrollEnabled = scrollEnabled;
   _webView.scrollView.scrollEnabled = scrollEnabled;
+
```

Comment:
Does this really comes from this PR?

Reply:
Looks like a merge issue.

https://github.com/react-native-community/react-native-webview/commit/2cef5cbb3ff5967a8f207b0c54340003ab1abc80